tests/test_algorithms_host_footprint_predict.py

```diff
@@ -0,0 +1,18 @@
+from networkml.algorithms.host_footprint_predict import predict
+
+def test_predict():
+    """
+    Test basic functionality of the prediction function -- Predict the
+    top business roles of a given device
+    """
+
+    ## Import training function to create inputs for predict function
+    from networkml.algorithms.host_footprint_training import train
```

Comment:
This should be moved to the top of the file with the other import statements

Reply:
Makes sense. I'll make that change. Thanks @cglewis